### PR TITLE
added image component.

### PR DESCRIPTION
**Pull request by zoyatanweer**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (zoef-yt) [inline .vscode/settings.json:3]: This file should be removed

**Review by zoef-yt — CHANGES_REQUESTED** [review]
Remove .vscode file. Else good to merge.

File: .vscode/settings.json (hunk 1 above)
Comment (varun13169) [inline .vscode/settings.json:3]: Add it in .gitignore and it'll never bug you again :)

File: components/image/image.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.round-img-xl, .round-img-lg{
+    margin:2rem;
+}
+.round{
+    border-radius:50% ;
+}
+.round-img-xl{
+    height:35rem;
```
Comment (varun13169) [inline components/image/image.css]: spacing between property and value will be better for readability.

**Review by varun13169 — CHANGES_REQUESTED** [review]

File: components/image/image.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            </li>
+            <li class="list-item">
+                <a class="link" href="https://github.com/zoyatanweer/falcon-ui">GITHUB</a>
+            </li>
+        </ul>
+    </nav>
+    <nav class="side-navigation-container">
+        <div class="nav header">
+            <h2>Getting Started</h2>
+            <div class="nav-content">
```
Comment (itsmebipul00) [inline components/image/image.html:31]: these look like a list you could use ul li then a

File: components/image/image.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+           Images can be round or square shaped depending upon the choice. They can also be made responsive to fit the parent's width.
+        </p>
+        <h2 class="sub-header">Round Image</h2>
+        <p class="section-about">Round shape images can be created by using the class <span>round-img- </span>and then using xl or lg as per required size.
+        <br>
+        <br>round image xl size = 35rem * 35rem
+        <br>round image lg size = 30rem * 30rem 
+        </p>
+        <div class="round-image">
+            <img class="round round-img-xl" src="/assets/images/ishan-shah-uEJZ7WGqHi4-unsplash.jpg">
```
Comment (itsmebipul00) [inline components/image/image.html]: alt tags on imgs

**Comment by itsmebipul00** [comment]
Nice pictures btw! Good to merge with changes

**Review by itsmebipul00 — APPROVED** [review]

File: components/image/image.css (hunk 2 above)
Comment (dev-prabhat) [inline components/image/image.css]: try to keep height mostly auto to make it responsive with respect to the container

**Review by dev-prabhat — CHANGES_REQUESTED** [review]

**Review by Omkar-Ghate — APPROVED** [review]
Looks great. Good to merge.

**Pull request merged**